config/composer/typo3-110-composer-packages-extensions.php

```diff
@@ -0,0 +1,376 @@
+<?php
```

Comment:
Again if I read this correctly, this cannot be right, there isn't that many extensions compatible with TYPO3 11.0 yet.

Reply:
True. The extension authors did something wrong which is difficult to catch. For example https://packagist.org/packages/dmk/mkpostman#1.0.3 is compatible regarding the composer.json definition. I have to find a better solution to prevent this. But didn´t come up with one yet.